packages/psss/src/containers/Tables/CountryReportTable.js

```diff
@@ -58,36 +87,57 @@ export const CountryReportTableComponent = React.memo(
 
     const handleSubmit = async () => {
       setTableStatus(TABLE_STATUSES.SAVING);
-      await onSubmit(fields);
+      await onSubmit({
+        ...fields,
+        sitesReported: parseInt(sitesReportedValue, 10),
+      });
       setTableStatus(TABLE_STATUSES.STATIC);
     };
 
     return (
       <LoadingContainer isLoading={tableStatus === TABLE_STATUSES.SAVING}>
-        <LayoutRow>
-          <Typography variant="h5">7/10 Sites Reported</Typography>
+        <FlexSpaceBetween pb={2}>
+          {tableStatus === TABLE_STATUSES.EDITABLE ? (
+            <FormRow>
+              <ReportedSites variant="h6">Reported Sites:</ReportedSites>
+              <StyledTextField
+                value={sitesReportedValue}
+                onChange={event => setSitesReportedValue(event.target.value)}
+                name="sites-reported"
+              />
+              <ReportedSites variant="h6"> / Total Sites: 10</ReportedSites>
+            </FormRow>
+          ) : (
+            <Typography variant="h5">7/{sitesReportedValue} Sites Reported</Typography>
+          )}
           <GreyOutlinedButton
             onClick={handleEdit}
             disabled={tableStatus === TABLE_STATUSES.EDITABLE}
           >
             Edit
           </GreyOutlinedButton>
-        </LayoutRow>
+        </FlexSpaceBetween>
+        {tableStatus === TABLE_STATUSES.EDITABLE && (
+          <Alert severity="error" variant="standard">
+            Updating aggregated data will be the source of truth. All individual Sentinel data will
```

Comment:
```suggestion
            Updating country level data manually: all individual sentinel site data will
```

Reply:
Dave suggested this red alert during edit, plus another grey alert that should show while the table is static, if the country level data has been manually edited - is that grey one captured in another card somewhere?

Reply:
I've updated the wording here to match the design update and added the grey alert ui component. There is still some work to do to finish the behaviour of the table (which data to show, when to show the alert and the reset button) but I think that is dependant on the backend and so I have created another card for it.